src/NovelRT.Interop/NrtInteropErrorHandlingInternal.cpp

```diff
@@ -8,8 +8,12 @@ const char* const errMsgIsNaN = "Unable to continue! Calculation resulted in an
 const char* const errMsgIsDivideByZero = "Unable to continue! Divide by zero was attempted.";
 const char* const errMsgIsNotSupportedInternal = "The current operation is not supported.";
 const char* const errMsgIsAlreadyDeletedOrRemoved = "Unable to continue! The specific item has already been deleted or removed.";
-const char* const errMsgIsInitialisationFailure = "Service could not be initialised.";
-const char* const errMsgIsFunctionNotFound = "Failed to locate the specified managed function.";
+const char* const errMsgIsInitialisationFailure = "Unable to continue! Service could not be initialised.";
+const char* const errMsgIsFunctionNotFound = "Unable to continue! Failed to locate the specified managed function.";
+const char* const errMsgIsNotInitialized = "Unable to continue! The service is currently not initialised.";
```

Comment:
US spelling of Initialised here also needs to be changed.

Reply:
KK